File: test/fixtures/benchmarks/ruby/bad_lint.rb
# Per-language badLint fixture, Ruby row.
# Recipe v3 (G4) scaffolded — adjust syntax / linter rules as needed.
# Deliberate RuboCop violations on default config:
#   - Lint/UselessAssignment (unused_var)
#   - Style/RedundantReturn (return at end of method)
def bad_lint
  unused_var = 42
  return 0
end

File: test/fixtures/benchmarks/php/bad_lint.php
// Per-language badLint fixture, PHP row.
// Recipe v3 (G4) scaffolded — adjust syntax / linter rules as needed.
<?php
// Deliberate PHP_CodeSniffer violations on the PSR-12 standard:
//   - opening brace on the declaration line (PSR12.Functions)
//   - uppercase TRUE/FALSE (Generic.PHP.LowerCaseConstant)
function bad_lint($flag){ if ($flag === TRUE){ return 1; } return FALSE; }
